ons_alpha/taxonomy/views.py

```diff
@@ -0,0 +1,63 @@
+# Create TopicViewSet
+from wagtail.admin.views.generic.chooser import ChooseResultsView, ChooseView
+from wagtail.admin.viewsets.chooser import ChooserViewSet
+from wagtail.admin.viewsets.model import ModelViewSet
+
+from .models import Topic
+
+
+class TopicChooserMixin:
+    def get_object_list(self):
+        # Only show top-level topics
+        return Topic.get_root_nodes()
+
+
+class SubTopicChooserMixin:
+    def get_object_list(self):
+        # Only show topic that have a parent
+        return Topic.objects.exclude(depth=1).order_by("path")
```

Comment:
I am not sure we need the extra complexity, especially that we can have more than 2 levels. In the screenshot I have a topic section under another topic section, 
<img width="948" alt="Screenshot 2024-07-16 at 14 32 00" src="https://github.com/user-attachments/assets/8177ae26-0d58-4dc4-b828-b9727f72873f">

but I only get the top level in the chooser:

<img width="474" alt="Screenshot 2024-07-16 at 14 32 08" src="https://github.com/user-attachments/assets/81820dd4-19fd-43c7-a928-e1f549227e28">

Reply:
I was thinking even "sub TopicSections" would be top level taxonomy Topics. But for now, I've removed the filtering and kept it simply.